### PR TITLE
Add obsloete->obsolete and friend

### PR DESCRIPTION
**Pull request by peternewman**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -32011,6 +32012,8 @@ obsevrer->observer
 obsevrers->observers
 obslete->obsolete
 obsleted->obsoleted
+obsloete->obsolete
+obsloeted->obsoleted
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:32016]: Perhaps add:
```suggestion
obsloescence->obsolescence
```

Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: I think you mean:
```suggestion
obsloeted->obsoleted
obsloescence->obsolescence
```

**Comment by DimitriPapadopoulos** [comment]
We might need to split the dictionary into multiple files. It's grown too big and GitHub has (transient for now) problems displaying the diffs.

**Comment by peternewman** [comment]
> We might need to split the dictionary into multiple files. It's grown too big and GitHub has (transient for now) problems displaying the diffs.

I'm pretty certain this is mostly #3017 but it has been getting bigger...

**Comment by DimitriPapadopoulos** [comment]
Could you run pre-commit locally and rebase into a single commit on top of _master_? Perhaps this will help in the short term.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**